packages/instant/src/util/asset_data.ts

```diff
@@ -0,0 +1,18 @@
+import { AssetProxyId } from '@0xproject/types';
+
+import { assetMetaData } from '../data/asset_meta_data';
+
+// TODO: tests for this
+export const bestNameForAsset = (assetData: string | undefined, defaultString: string) => {
+    if (assetData === undefined) {
+        return defaultString;
+    }
+    const metaData = assetMetaData[assetData];
+    if (metaData === undefined) {
+        return defaultString;
+    }
+    if (metaData.assetProxyId === AssetProxyId.ERC20) {
+        return metaData.symbol.toUpperCase();
```

Comment:
technically yes, I thought about doing this: I generally like using textTransform for displaying things in all uppercase, But, this function would have to return a JSX/HTML element instead of just a string.  I prefer the simplicity of the way it is now, but I could be convinced if you feel strongly